shims/dataproc1421/driver/pom.xml

```diff
@@ -13,6 +13,8 @@
 
   <properties>
     <netty.version>4.1.108.Final</netty.version>
+    <!-- settting v1.14.4 so shaded jackson-databind inside parquet-hadoop-bundle's  version approximately matches jackson-databind in driver.jar -->
+    <parquet.version>1.14.4</parquet.version>
```

Comment:
dataproc1421 shim doesn't have a parquet-pid dependency, not sure why. main goal is to address parquet-hadoop-bundle